test/requirements.txt

```diff
@@ -4,213 +4,367 @@
 #
 #    pip-compile --output-file=requirements.txt requirements.in
```

Comment:
You should just run this command with the new requirement added to `requirements.in`, not with `--upgrade`. Also make sure to restore this `requirements.txt` from the main branch before you do that.

Reply:
Followed the exact same steps now and generated the requirements.txt file. Marking this as resolved.